.gitlab-ci.yml

```diff
@@ -1352,3 +1352,17 @@ workflow:
 .on_fakeintake_changes_on_main:
   - <<: *on_fakeintake_changes
     if: $CI_COMMIT_BRANCH == "main"
+
+.on_invoke_tasks_changes:
+  - !reference [.except_mergequeue]
+  - changes:
+      paths:
+        - tasks/**/*
+      compare_to: main # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
```

Comment:
```suggestion
    paths:
      - tasks/**/*
    compare_to: main # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
```

Reply:
This is the current indentation use elsewhere in the file, I'll stick to it right now